ReportSimulationOutput/measure.rb

```diff
@@ -1208,7 +1208,7 @@ def check_for_errors(runner, outputs)
     # Check if simulation successful
     all_total = @fuels.values.map { |x| x.annual_output.to_f }.sum(0.0)
     all_total += @ideal_system_loads.values.map { |x| x.annual_output.to_f }.sum(0.0)
-    if all_total == 0
+    if all_total == 0 && (not @hpxml.header.power_outage_periods.any? { |op| op.year_round? })
```

Comment:
For year round outages, you'd expect `all_total == 0` to evaluate to true.

Reply:
I don't think we really expect to ever get a year-round power outage. Why would someone want to simulate that just to get zero energy use?

Reply:
Year-round power outages facilitate testing. Should I delete all the year-round HPXML test files?

Reply:
I delete them. And the check on `year_round?`.